doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
And, I was wrong. I forgot to *invoke* epipebomb. It totally swallows the EPIPE error as advertised.